scripts/compile-mlibc-sysdeps.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+MLIBC_PATH=$1
+KERNEL_HEADER_PATH=$2
+shift 2
+SYSDEPS=${*:-"dripos lemon aero ironclad vinix lyre"}
```

Comment:
redundant variable, just use "$@" in the for loop, or just `for x; do`, since that iterates over $@ anyway